Gemfile

```diff
@@ -23,7 +23,7 @@ gem 'daemons' # ruby code can be run as daemon with simple start/stop/restart co
 gem 'whenever', :require => false
 
 gem 'druid-tools'
-gem 'is_it_working-cbeer'
+gem 'okcomputer'
```

Comment:
we can only remove is_it_working in the same PR if  nagios isn't expecting anything at `is_it_working` path.